ironic/conf/drac.py

```diff
@@ -21,7 +21,14 @@
                min=1,
                help=_('Interval (in seconds) between periodic RAID job status '
                       'checks to determine whether the asynchronous RAID '
-                      'configuration was successfully finished or not.'))
+                      'configuration was successfully finished or not.')),
+    cfg.IntOpt('query_management_config_job_status_interval',
```

Comment:
If we remove the periodic job, then we won't need this setting.

Reply:
Done